easy_phi_client.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import json
+import urllib
+import logging
+
+import settings
```

Comment:
whenever you have external settings file, you create a dependency. It HAS to be passed from outside, e.g. from console parameter or environment variable. Also, you need to provide reasonable default (e.g. localhost)